acceptance/ui/features/applications.feature

```diff
@@ -0,0 +1,161 @@
+@apps
+Feature: Application Management
+  In order to use Control Center
+  As a CC admin user
+  I want to manage applications
+
+  @login-required
+  Scenario: View default applications page
+    When I am on the applications page
+    Then I should see "Application Template"
+      And I should see "Deployment ID"
+      And I should see "Virtual Host Names"
+      And I should see "Internal Services"
+      And I should see "Description"
+      And I should see "Services Map"
+
+  @run @login-required
+  Scenario: View services map
+    When I am on the applications page
+      And I click the Services Map button
+    Then I should see "Internal Services" in the Services Map
+
+  @login-required
+  Scenario: View application template dialog
+    When I am on the applications page
+      And I click the Add-Application Template button
+    Then I should see "Add Application Template"
+      And I should see "Application Template File:"
+      And I should see "No file chosen"
```

Comment:
When I ran this test on my machine, the dialog contained "No file selected" instead of "No file chosen". 
Did does it display on your test?